src/bandersnatch/utils.py

```diff
@@ -212,3 +213,9 @@ def parse_version(version: str) -> List[str]:
     )
 
     return _versions
+
+
+def write_fake_json(json_path: Path) -> None:
+    if not json_path.parent.exists():
+        json_path.parent.mkdir(parents=True)
+    json_path.write_text(json.dumps({"releases": {}}))
```

Comment:
```suggestion
    json_path.write_text(json.dumps({}))
```
Might as well make it as small as possible ...